applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -780,6 +787,19 @@ static int init(void)
 		cur_peer_id_valid = true;
 	}
 
+	if (IS_ENABLED(CONFIG_DESKTOP_BT_PERIPHERAL)) {
+		/* Ensure that there are no peers bonded to temporary peer Bluetooth identity.
+		 * A bonded peer may not be properly removed e.g. in case of spurious power down.
+		 */
+		int err = bt_unpair(bt_stack_id_lut[TEMP_PEER_ID], NULL);
+
+		if (err) {
+			LOG_ERR("Cannot unpair for temporary peer Bluetooth identity: %" PRIu8,
+				bt_stack_id_lut[TEMP_PEER_ID]);
+			module_set_state(MODULE_STATE_ERROR);
+		}
```

Comment:
The temporary Bluetooth local identity is not assigned to any application local identity, so that we cannot use application local identity when referring to it.